File: SecondSemester/OOP/Lab5/C#/SymbolString.cs
using System;

namespace Lab5
{
    public class SymbolString : String
    {
        public SymbolString(string str) : base(str)
        {
            
        }
        public void Sort()
        {
            char[] temp = Sentence.ToCharArray();
            Array.Sort(temp);
            Array.Reverse(temp);
            Sentence = new string(temp);
        }
    }
}

File: SecondSemester/OOP/Lab5/C++/SymbolString.cpp
//
// Created by bohdan on 07.04.20.
//

#include "SymbolString.h"

SymbolString::SymbolString(string str) : String(str)
{
}

void SymbolString::Sort() {
    std::sort(Sentence.begin(), Sentence.end());
    std::reverse(Sentence.begin(), Sentence.end());
};